test/AspNet.Security.OAuth.Providers.Tests/Zalo/ZaloTests.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System.Security.Claims;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authentication;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace AspNet.Security.OAuth.Zalo
+{
+    public class ZaloTests : OAuthTests<ZaloAuthenticationOptions>
+    {
+        public ZaloTests(ITestOutputHelper outputHelper)
+        {
+            OutputHelper = outputHelper;
+        }
+
+        public override string DefaultScheme => ZaloAuthenticationDefaults.AuthenticationScheme;
+
+        protected internal override void RegisterAuthentication(AuthenticationBuilder builder)
+        {
+            builder.AddZalo(options => ConfigureDefaults(builder, options));
+        }
+
+        [Theory]
+        [InlineData(ClaimTypes.NameIdentifier, "my-id")]
+        [InlineData(ClaimTypes.Name, "HoangITK")]
```

Comment:
You should add a test case for the `DateOfBirth` and `Gender` claims as you're adding code to deal with them.